releasenotes/notes/add-xxminusyy-gate-63e6530c23500de9.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Adds a gate for the XX-YY interaction, accessible via
+    `qiskit.circuit.library.XXMinusYYGate`
```

Could you maybe add a sentence mentioning why we're adding this gate? E.g. because it's frequently used for a certain application in chemistry or so
```suggestion
   :class:`~qiskit.circuit.library.XXMinusYYGate`.
```